sections/selector.tex

```diff
@@ -1,7 +1,15 @@
 \section{Writing a Selector}\label{sec:selector}
 
-The lepton selection we want to implement in this exercise is the following:
+The \CCSPStlye{Selector} class should be used to implement analysis selections.
+This is a crucial step in the workflow since the decision to keep or reject objects or even whole events is performed here.
+Since the selection usually depends on for example four momenta of the objects within the events, it is executed after the calibration.
```

Comment:
maybe use "four-momenta" like in [sections/calibrator.tex](https://github.com/columnflow/demo_docs/pull/1/files#diff-48300489eb6890d1807b94ce19b255755b700dc03370d987fba8fd4c2dfce4cf)